<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>411</width>
      <height>131</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QSpinBox {
  background-color: white;
  color: black; 
  border: 1px solid black;
}
</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>0</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(0,0,255);
  color: white; 
  border: 1px solid black;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
      </property>
      <property name="text">
       <string>Добавить 
 в очередь</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>0</y>
        <width>91</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(0, 0, 255);
  color: white; 
  border: 1px solid black;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
      </property>
      <property name="text">
       <string>Печать 
 элеметов</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>50</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(0,0,255);
  color: white; 
  border: 1px solid black;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
      </property>
      <property name="text">
       <string>Удалить 
 из очереди</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
  background-color: white;
  color: black; 
  border: 1px solid black;
}
</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
  background-color: white;
  color: black; 
  border: 1px solid black;
}
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
  background-color: white;
  color: black; 
  border: 1px solid black;
}
</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>0</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid black;
}

QPushButton:pressed {
    background-color:red ;
}
</string>
      </property>
      <property name="text">
       <string>Добавить 
 в очередь</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>50</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid black;
}

QPushButton:pressed {
    background-color:red ;
}</string>
      </property>
      <property name="text">
       <string>Удалить 
 из очереди</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>0</y>
        <width>91</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid black;
}

QPushButton:pressed {
    background-color:red ;
}</string>
      </property>
      <property name="text">
       <string>Печать 
 элеметов</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
